### DISCUSSION TITLE
News: Integração com Telegram

### DISCUSSION THREAD

**Discussion by eduardojmatos** [frontendbr/forum#597] [Discussão]
Pessoal, agora temos integração com Telegram, pra quem curte. As notificações vão todas pra esse canal: [https://t.me/frontendbr_forum](https://t.me/frontendbr_forum).

Valeu tio @suissa pela dica e pelo setup!!!

**Comment by suissa** [comment]
@eduardojmatos Respondendo via Telegram na altaaaa!

**Comment by felipefialho** [comment]
Boa!

**Comment by ghost** [comment]
@LFeh he did a crusade... lmao
